go.mod

```diff
@@ -1,17 +1,23 @@
 module tinygo.org/x/bluetooth
 
-go 1.15
+go 1.18
 
 require (
 	github.com/go-ole/go-ole v1.2.6
-	github.com/godbus/dbus/v5 v5.0.3
-	github.com/muka/go-bluetooth v0.0.0-20220830075246-0746e3a1ea53
+	github.com/godbus/dbus/v5 v5.1.0
+	github.com/muka/go-bluetooth v0.0.0-20221213043340-85dc80edc4e1
 	github.com/saltosystems/winrt-go v0.0.0-20230510070731-e096b9afa761
-	github.com/sirupsen/logrus v1.9.0 // indirect
 	github.com/tinygo-org/cbgo v0.0.4
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
-	golang.org/x/sys v0.0.0-20220829200755-d48e67d00261 // indirect
-	golang.org/x/text v0.5.0
 	tinygo.org/x/drivers v0.23.0
 	tinygo.org/x/tinyterm v0.1.0
 )
+
+require (
+	github.com/fatih/structs v1.1.0 // indirect
+	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
+	github.com/sirupsen/logrus v1.6.0 // indirect
```

Comment:
Funny enough this indirect dependency become downgraded. And this is actually correct, since `muka` explicitly depends on `v1.6.0` of `logrus`, so using newer version probably not wise anyway.